public/app/plugins/datasource/grafana-azure-monitor-datasource/resource_picker/resource_picker_data.ts

```diff
@@ -0,0 +1,168 @@
+import { getBackendSrv } from '@grafana/runtime';
+import { EntryType, Row, RowGroup } from '../components/ResourcePicker/types';
+import produce from 'immer';
+
+const RESOURCE_GRAPH_URL =
+  '/resourcepickerdata/providers/Microsoft.ResourceGraph/resources?api-version=2020-04-01-preview';
+
+export default class ResourcePickerData {
+  private _proxyUrl: string;
+  private _formattedResourcePickerData: RowGroup;
```

Comment:
not sure what's the policy here, but in the past I've used this linter rule  (https://eslint.org/docs/rules/no-underscore-dangle) that prevents people from using underscores in var names (since there is no need to do so).

Reply:
yeah we don't use underscores to mark it's private. see [frontend styleguide](https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md) (aka airbnb). 
that eslint rule would be nice to have!